### ISSUE TITLE
Add "Only update on WiFi" option

### ISSUE THREAD

**Issue by rock3r** [frakbot/FWeather#53]
This is to save data and power.

**Comment by blchinezu** [comment]
Either this way or simply use the existing "Sync frequency" option. So there would be two options "3G Sync Frequency" and "WiFi Sync Frequency". This would also give the possibility of setting 3 hours for mobile networks and 10 minutes for WiFi (just as an example). I don't know how useful would that be but I would certainly use it.

**Comment by rock3r** [comment]
I don't really see having two separate sync timers as something useful for
anyone else, I'm afraid :) I think "update on WiFi only" is even way too
much of a choice for the vast majority of the users...

**Comment by blchinezu** [comment]
Fair enough. I wasn't expecting someone to use this except me anyway. :))

**Comment by rock3r** [comment]
You can always fork the app :D
